ui/raidboss/data/05-shb/trial/varis-ex.js

```diff
@@ -0,0 +1,124 @@
+'use strict';
+[{
+  zoneRegex: {
+    en: /^Memoria Misera \(Extreme\)$/,
+  },
+  timelineFile: 'varis-ex.txt',
+  triggers: [
+    {
+      id: 'VarisEx Phase 2',
+      regex: Regexes.startsUsing({ source: 'Varis Yae Galvus', id: '4CCC', capture: false }),
+      run: function(data) {
+        data.phase = 'phase2';
+      },
+    },
+    {
+      id: 'VarisEx Ignis Est',
+      regex: Regexes.startsUsing({ source: 'Ignis Est', id: '4CB6', capture: false }),
+      delaySeconds: 2,
+      response: Responses.getOut(),
+    },
+    {
+      id: 'VarisEx Ventus Est',
+      regex: Regexes.startsUsing({ source: 'Ventus Est', id: '4CC7', capture: false }),
+      delaySeconds: 2,
+      response: Responses.getIn('info'),
+    },
+    {
+      id: 'VarisEx Altius',
+      regex: Regexes.startsUsing({ source: 'Varis Yae Galvus', id: '4CCA', capture: false }),
+      infoText: {
+        en: 'Bait Slashes',
+      },
+    },
+    {
+      id: 'VarisEx Citius',
+      regex: Regexes.startsUsing({ source: 'Varis Yae Galvus', id: '4CF0' }),
+      alertText: function(data, matches) {
+        if (data.me == matches.target) {
+          return {
+            en: 'Buster on YOU',
+          };
+        }
+        if (data.role == 'dps') {
+          return {
+            en: 'Avoid Tanks',
+          };
+        }
+        return {
+          en: 'Buster on ' + data.ShortName(matches.target),
+        };
+      },
+    },
+    {
+      id: 'VarisEx Alea lacta Est',
+      regex: Regexes.startsUsing({ source: 'Varis Yae Galvus', id: '4CD2', capture: false }),
+      response: Responses.getBehind('alarm'),
```

Comment:
Thanks for changing this to differ getBehind/goFront, but I think there are too many "alarms" in this fight.  Can getBehind be info (because most people already behind) and go front be alert (since more unusual)?

See also: https://github.com/quisquous/cactbot/blob/master/CONTRIBUTING.md#trigger-severity

Reply:
I think at least alert is warranted for get behind, as this attack happens immediately after the boss turns towards a random player, so most people are actually in front when this attack starts.

Reply:
I agree with alert for `getBehind()`. The boss can be baited into facing a particular direction, but failure to move means the user will most likely die. (If they are not already moving by the time the animation starts, they will be hit at least twice.)